src/components/action-sheet/action-sheet.ios.scss

```diff
@@ -97,16 +97,15 @@ $action-sheet-ios-button-cancel-font-weight:            600 !default;
 
 .action-sheet-ios .action-sheet-group {
   @include border-radius($action-sheet-ios-border-radius);
+  @include margin(null, null, $action-sheet-ios-group-margin-bottom - 2, null);
```

Comment:
Why do we need to use the mixin for `margin-bottom`, `margin-top`, `padding-bottom`, or `padding-top`? These shouldn't be affected by RTL, no?